stdlib/tkinter/__init__.pyi

```diff
@@ -3021,7 +3024,27 @@ class Text(Widget, XView, YView):
     config = configure
     def bbox(self, index: _TextIndex) -> tuple[int, int, int, int] | None: ...  # type: ignore[override]
     def compare(self, index1: _TextIndex, op: Literal["<", "<=", "==", ">=", ">", "!="], index2: _TextIndex) -> bool: ...
-    def count(self, index1, index2, *args): ...  # TODO
+    @overload
+    def count(self, index1: _TextIndex, index2: _TextIndex) -> tuple[int] | None: ...
```

Comment:
I intentionally made these positional-or-keyword, because it is technically possible to do `text.count(index1=foo, index2=bar)`. It is IMO not any better than `.count(foo, bar)`, but it works, so we might as well support it.